torchtitan/config_manager.py

```diff
@@ -299,14 +299,14 @@ def __init__(self):
         self.parser.add_argument(
             "--experimental.pipeline_parallel_schedule",
             type=str,
-            choices=["1f1b", "gpipe", "interleaved_1f1b", "flexible_interleaved_1f1b"],
+            choices=["1f1b", "gpipe", "Interleaved1F1B", "FlexibleInterleaved1F1B"],
```

Comment:
any reason we leave `1f1b` and `gpipe` in lower cases?

Reply:
The `get_schedule()` method is case insensitive, the main change was i needed to remove the "_" in the other schedules. But I can change these as well for consistency.